deploy/grafana/grafana-with-grafana-operator/install-grafana-with-operator.sh

```diff
@@ -57,21 +63,68 @@ kubectl apply --namespace="${GRAFANA_NAMESPACE}" -f <( \
     envsubst \
 )
 
-echo "Waiting to start"
-sleep 60
+echo "Waiting to start Grafana custom resource $GRAFANA_NAME"
+sleep 10
 
 kubectl apply --namespace="${GRAFANA_NAMESPACE}" -f <( \
-    cat grafana-data-source-cr-template.yaml | \
-    GRAFANA_DATA_SOURCE_NAME="$GRAFANA_DATA_SOURCE_NAME" \
+    cat ${CUR_DIR}/grafana-data-source-cr-template.yaml | \
+    GRAFANA_PROMETHEUS_DATASOURCE_NAME="$GRAFANA_PROMETHEUS_DATASOURCE_NAME" \
     PROMETHEUS_URL="$PROMETHEUS_URL" \
     envsubst \
 )
 
-echo "Waiting to start"
-sleep 60
+echo "Waiting to apply GraganaDatasource custom resource $GRAFANA_PROMETHEUS_DATASOURCE_NAME"
+sleep 10
 
 kubectl apply --namespace="${GRAFANA_NAMESPACE}" -f <( \
-    cat grafana-dashboard-cr-template.yaml | \
+    cat ${CUR_DIR}/grafana-dashboard-cr-template.yaml | \
     GRAFANA_DASHBOARD_NAME="$GRAFANA_DASHBOARD_NAME" \
+    GRAFANA_PROMETHEUS_DATASOURCE_NAME="$GRAFANA_PROMETHEUS_DATASOURCE_NAME" \
     envsubst \
 )
+
+echo "Waiting to apply vertamedia-clickhouse-datasource plugin"
+sleep 10
```

Comment:
Timeouts are system specific. E.g. if image should be downloaded, then it may be more than 10 seconds.
Please add a true wait loop checking for certain resource to be available.